docs/src/ExpandableModule.doc.js

```diff
@@ -0,0 +1,162 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { ExpandableModule } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards: Array<Node> = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="ExpandableModule"
+    description="
+The ExpandableModule component renders detailed information in the expanded state and summarized information in the collapsed state.
+"
+  />
+);
+
+card(
+  <PropTable
+    Component={ExpandableModule}
+    props={[
+      {
+        name: 'id',
+        type: 'string',
+        required: true,
+      },
+      {
+        name: 'accessibilityExpandLabel',
+        type: 'string',
+        required: true,
+      },
+      {
+        name: 'accessibilityCollapseLabel',
+        type: 'string',
+        required: true,
+      },
+      {
+        name: 'items',
+        type:
+          'Array<{| title: string, icon?: $Keys<typeof icons>, summary?: Array<string>, type?: "info" | "error", iconAccessibilityLabel?: string, children: ?React.Node |}>',
+        required: true,
+      },
+    ]}
+  />
+);
+
+card(
+  <Example
+    name="Example"
+    defaultCode={`
+function ExpandableModuleExample1() {
+  return (
+    <Box maxWidth={800} padding={2} column={12}>
+      <ExpandableModule
+        id={expandableUniqueID}
```

Comment:
@galaxy0101 this should be a string `id="ExpandableModuleExample1"`

Reply:
Fixed.